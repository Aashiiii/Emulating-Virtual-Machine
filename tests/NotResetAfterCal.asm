FUNC LABEL 0
    MOV STK A VAL 0
    MOV REG 0 VAL 2
    MOV REG 1 STK A
    PRINT STK A
    PRINT REG 0
    CAL VAL 3
    RET
FUNC LABEL 3
    ADD REG 0 REG 1
    PRINT REG 0
    PRINT STK A
    RET
