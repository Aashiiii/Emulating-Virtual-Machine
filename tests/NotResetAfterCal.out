0
2
2
119
